src/contrib/persistence/Akka.Persistence.Sql.Common/Journal/QueryExecutor.cs

```diff
@@ -628,8 +628,8 @@ protected DbCommand GetCommand(DbConnection connection, string sql)
         protected virtual void WriteEvent(DbCommand command, IPersistentRepresentation e, IImmutableSet<string> tags)
         {
             var manifest = string.IsNullOrEmpty(e.Manifest) ? e.Payload.GetType().TypeQualifiedName() : e.Manifest;
-            var serializer = Serialization.FindSerializerFor(e.Payload);
-            var binary = serializer.ToBinary(e.Payload);
+            var serializer = Serialization.FindSerializerForType(typeof(IPersistentRepresentation));
```

Comment:
The main idea of all SQL plugins is to store payload only into `Payload` column, not the whole message. We are using columns to store the header information, like `PersistenceId`, `SequenceNr`. So, we don't need any Protobuf serializers there, because those serializers don't serialize a payload.